docs/editor_manual/administrator_tasks/managing_users.md

```diff
@@ -0,0 +1,34 @@
+# Managing users and roles
+
+As an administrator, a common task will be adding, modifying or removing user profiles.
+
+This is done via the 'Users' interface, which can be found in the Settings menu, accessible via the left-hand menu bar.
+
+```{image} ../../_static/images/screen35_users_menu_item.png
```

Comment:
All images can probably use markdown syntax 
```
![](../../_static/images/screen35_users_menu_item.png)
```

Reply:
It might be nice to add an alt text (the content between the `[]` while we are at it.